docs/connection.md

```diff
@@ -121,43 +121,30 @@ def send(self, envelope: Envelope):
 -->
 
 
+## Launching `oef` connections
 
-## Envelope
+### `oef` - local node
 
-An `Envelope` wraps messages. It travels from `OutBox` to another agent. `Envelope` objects sent from other agents arrive in the `InBox` via a protocol connection.
-
-An `Envelope` objects has four instance variables.
+Download the scripts directory:
+``` bash
+svn export https://github.com/fetchai/agents-aea.git/trunk/scripts
+```
 
-* `to`: an `Address` object defining the recipient agent.
-* `sender`: an `Address` object defining the sender agent.
-* `protocol_id`: the id of the protocol.
-* `message`: the message in bytes.
+Then, start an oef from a separate terminal:
```

Comment:
> Then, run the following command from a separate terminal

Because it is not an OEF Node or an oef. It is just a python script that the user is going to run in a different terminal. And you explain after the command what it actually does.